### PR TITLE
Add Notification Metrics dashboard

### PR DESCRIPTION
**Pull request by devinbinnie**
#### Summary
This adds the Notification Metrics dashboard for upload to Grafana

#### Ticket Link
https://mattermost.atlassian.net/browse/MM-58590

### CODE REVIEW

File: grafana/mattermost-notification-metrics.json [hunk 1]
```diff
@@ -0,0 +1,1310 @@
...
+        {
+          "id": "calculateField",
+          "options": {
+            "alias": "",
+            "binary": {
+              "left": "mattermost_notifications_total_ack{instance=\"host.docker.internal:8067\", job=\"mattermost\", type=\"websocket\"}",
+              "operator": "/",
+              "reducer": "sum",
+              "right": "mattermost_notifications_total{instance=\"host.docker.internal:8067\", job=\"mattermost\", type=\"websocket\"}"
+            },
```
Comment (agnivade) [inline grafana/mattermost-notification-metrics.json]: This seems to have unwanted labels like `job` that I was talking about. And also I don't think we want to hardcode to the docker internal host.

File: grafana/mattermost-notification-metrics.json [hunk 2]
```diff
@@ -0,0 +1,1310 @@
...
+      "transformations": [
+        {
+          "id": "calculateField",
+          "options": {
+            "alias": "",
+            "binary": {
+              "left": "mattermost_notifications_total_ack{instance=\"host.docker.internal:8067\", job=\"mattermost\", type=\"websocket\"}",
+              "operator": "/",
+              "reducer": "sum",
+              "right": "mattermost_notifications_total{instance=\"host.docker.internal:8067\", job=\"mattermost\", type=\"websocket\"}"
```
Comment (agnivade) [inline grafana/mattermost-notification-metrics.json]: Same as earlier

**Review by agnivade — CHANGES_REQUESTED** [review]

File: grafana/mattermost-notification-metrics.json [hunk 3]
```diff
@@ -0,0 +1,1273 @@
...
```
Comment (agnivade) [inline grafana/mattermost-notification-metrics.json]: I am still seeing these. Is this intended?

Comment (devinbinnie) [reply grafana/mattermost-notification-metrics.json]: Huh, I didn't realize how many places I added these. Will remove the rest.

**Review by agnivade — APPROVED** [review]

**Pull request merged**